keyboards/cospad/cospad.c

```diff
@@ -16,18 +15,9 @@
  */
 #include "cospad.h"
 
-#ifdef BACKLIGHT_ENABLE
-
-void backlight_init_ports(void) {
-  setPinOutput(F7);
-}
-
-void backlight_set(uint8_t level) {
-  writePin(F7, !!level);
+bool led_update_kb(led_t led_state) {
```

Comment:
Tested on my board, doesnt work without...
```suggestion
void keyboard_pre_init_kb(void) {
    setPinOutput(B2);

    keyboard_pre_init_user();
}

bool led_update_kb(led_t led_state) {
```

Reply:
I knew there was something I forgot...